opencog/relex/Dockerfile

```diff
@@ -0,0 +1,98 @@
+#
+# Docker file that builds RelEx and starts the RelEx server.
+#
```

Comment:
Hmm. The relex dockerfile comes with relex ... having ttwo copies will lead to heartache ... ... perhaps there could be a build.sh file that gets githubusercontent/raw/relex/Dockerfile, and builds that?